src/app/info/WikiItemEdit.tsx

```diff
@@ -24,14 +33,20 @@ const WikiItemEdit: React.FC<WikiItemEditProps> = ({
     setIsInEditMode,
     appendNewRow,
     removeRow,
+    swapRows,
+    setErrorMessage,
 }) => {
     const deleteWikiItem = async (): Promise<void> => {
         const deleteResponse = (await supabase
             .from("wiki")
             .delete()
             .match({ wiki_key: rowData.wiki_key })) as WikiRowQueryType;
         if (deleteResponse.error) {
-            logErrorReturnLogId("error deleting wiki row item", deleteResponse.error);
+            const logId = await logErrorReturnLogId(
+                "error deleting wiki row item",
+                deleteResponse.error
+            );
+            setErrorMessage(`Failed to delete wiki item. Log ID: ${logId}`);
         }
         removeRow(rowData);
         setRowData(undefined);
```

Comment:
I know this wasn't your work, but can we change this so we only remove the row if the database deletion succeeds?